src/modules/75-cf/components/TargetManagementFlagsListing/FlagDetailsCell.tsx

```diff
@@ -7,24 +7,44 @@
 
 import React, { FC } from 'react'
 import { Text } from '@harness/uicore'
-import { FontVariation } from '@harness/design-system'
+import { Color, FontVariation } from '@harness/design-system'
+import { Link, useParams } from 'react-router-dom'
 import type { Feature } from 'services/cf'
+import useActiveEnvironment from '@cf/hooks/useActiveEnvironment'
+import routes from '@common/RouteDefinitions'
 
 export interface FlagDetailsCellProps {
   row: { original: Feature }
 }
 
 export const FlagDetailsCell: FC<FlagDetailsCellProps> = ({
   row: {
-    original: { name, description }
+    original: { identifier, name, description }
   }
-}) => (
-  <>
-    <Text lineClamp={1} font={{ variation: FontVariation.BODY2 }}>
-      {name}
-    </Text>
-    {description && <Text lineClamp={2}>{description}</Text>}
-  </>
-)
+}) => {
+  const { projectIdentifier, orgIdentifier, accountId: accountIdentifier } = useParams<Record<string, string>>()
+  const { withActiveEnvironment } = useActiveEnvironment()
+
+  return (
+    <>
+      <Link
+        target="_blank"
+        to={withActiveEnvironment(
+          routes.toCFFeatureFlagsDetail({
+            featureFlagIdentifier: identifier,
+            orgIdentifier: orgIdentifier as string,
+            projectIdentifier: projectIdentifier as string,
```

Comment:
I'm pretty sure `as string` isn't needed since it's getting its type from `Record<string, string>`
```suggestion
            orgIdentifier: orgIdentifier,
            projectIdentifier: projectIdentifier,
```

Reply:
ah good point thank you I have updated :)